src/db/collections/outputs/mod.rs

```diff
@@ -96,6 +99,7 @@ impl From<&LedgerOutput> for OutputDocument {
                 is_trivial_unlock,
                 rent_structure,
             },
+            raw: rec.raw.clone(),
```

Comment:
❌  I don't think this is a good idea. We can just pack the outputs.

Reply:
you mean at query time?

Reply:
I agree with @Alexandcoats

Reply:
I agree transitively.

Reply:
😁